Client/components/sidebarsections.jsx

```diff
@@ -10,10 +10,9 @@ const SidebarSections = (props) => {
     }
 
     return (
-    <div>
-      <span style={{position: 'absolute', fontSize: 20, color: 'white'}}>{props.sectionInfo}</span>
-      <span style={{position: 'absolute', opacity: '35%', width: '200px', height: '167px', backgroundColor: 'white'}}></span>
-      <img src={photo} style={{ height: '167px' }}/>
+    <div style={{ position: 'relative', width: '200px', height: '167px', border: '1px solid white', marginTop: '0px', backgroundColor: 'rgba(74,74,74,.6)'}}>
```

Comment:
With regard to mixing in the styling here similar to mentioned above.
There's a lot of in-line style which does mean the state of the app is a little less visible. If you had the styling more isolated to your styled components, you can keep your component with your model data (props) a little cleaner.

Otherwise, I'd recommend you at least put each attribute on its own line:

```
<img src={photo} 
          style={{ width: '100%', height: '100%', opacity: '35%' }}
/>
```

Example: https://github.com/prettier/prettier/issues/5501

Its easier for another developer to visually scan the component and separate model data like props and styles.

Reply:
Yeah, that makes sense. My goal is to get as many of them transitioned into styled components as possible, but the main issue is that it freaked out when I had a component return as a styled component from another page. Because of that, I temporarily switched to some inline styling when nested, but have started to see which need to be nested and which don't, and the later will be styled components.

Reply:
Thanks for the info!